frontend/src/styles/about_us/ValueCard.module.css

```diff
@@ -97,3 +97,11 @@
     opacity: 1;
   }
 }
+
```

Comment:
Good--but actually our mobile styles are the styles by default. So to add them, you don't need a media query/just adding them at the top would have done the job. 

Sorry it wasn't explained somewhere else >w<